### DISCUSSION TITLE
Questions about Infrastructure/Deployment choices

### DISCUSSION THREAD

**Discussion by macel94** [microsoft/dotnet-podcasts#8] [Q&A] [answered]
Hello, 

If possible, I just wanted @nishanil to clarify 2 choices:
1) For CI/CD, Why did you choose github workflows and not Azure DevOps Pipelines?
2) You guys mentioned this sample is meant to be simple, shouldn't it be better\clearer to have bicep files for IAC instead of ARM json templates?

Thank you anyway for your help & guidance, I will try the sample and study how you guys structured things.

**Comment by nishanil** [comment, marked as answer]
Hey @MACEL94, Great questions! 
 1.  We chose GitHub Actions because the code is in GitHub already, making it easier for everyone to fork and set up their own workflows. 
 2.  We do have a bicep file [for the API deployment](https://github.com/microsoft/dotnet-podcasts/blob/main/deploy/Services/api.deployment.bicep). We started with ARM jsons and got it all working. Since we were in a hurry to release, we didn't get a chance to modify them all. 😊 Feel free to send us a PR Or I am sure @borjasanes can spin this around in no time 😉
